app/controllers/manage_evaluators_controller.rb

```diff
@@ -29,7 +29,7 @@ def create
   end
 
   def destroy
-    @phase = @challenge.phases.find(params[:phase_id])
+    @phase = Phase.find(params[:phase_id])
```

Comment:
scope to the current_user's challenge phase to prevent a challenge manager from deleting other phases
```suggestion
    @phase = @challenge.phases.find(params[:phase_id])
```

Reply:
@stepchud Just updated those and the tests, thanks!